.circleci/config.yml

```diff
@@ -1,36 +1,26 @@
 # Cache key for CircleCI. We want to invalidate the cache whenever the Bazel workspace or the
 # NPM dependencies changed.
 var_1: &cache_key angular-tsickle-{{ checksum "WORKSPACE" }}-{{ checksum "yarn.lock" }}
-# Default docker image for CircleCI jobs with Bazel installed. The version of Bazel is controlled
```

Comment:
Are these changes just cleanup, or in some way load bearing?

Reply:
cleanup, see the three commits here, the last one is just the refactoring